include/zephyr/device.h

```diff
@@ -223,7 +224,9 @@ typedef int16_t device_handle_t;
  *
  * @return A pointer to the device object created for that node
  */
-#define DEVICE_DT_GET(node_id) (&DEVICE_DT_NAME_GET(node_id))
+#define DEVICE_DT_GET(node_id) \
```

Comment:
... _OR_NULL for consistency, no?

Reply:
this change actually makes the _OR_NULL obsolete. We just have one that may return NULL. Compile error is now generated by the usage of DEVICE_DT_DECLARE.